app/javascript/oldjs/controllers/timeline/timeline_options_controller.js

```diff
@@ -7,6 +7,7 @@ ManageIQ.angular.app.controller('timelineOptionsController', ['miqService', 'url
       tl_timerange: 'weeks',
       tl_timepivot: 'ending',
       tl_result: 'success',
+      tl_storage_systems: [''],
```

Comment:
Is the rest of this provider specific?  I don't see anything else here about infra hosts or cloud vms...

Reply:
This is storage specific, is there a better solution in your eyes?